src/DataService/Media/Chapter.php

```diff
@@ -87,7 +87,7 @@ public function getThumbnailUrl(): Uri
     /**
      * Set a link to the thumbnail image for this chapter.
      *
-     * @param Uri
+     * @param Uri $thumbnailUrl The URL of the thumbnail.
```

Comment:
Should the description go in the next line?

Reply:
I'm following the Symfony style guides for the library. See [.php_cs.dist](https://github.com/Lullabot/mpx-php/blob/master/.php_cs.dist).

Reply:
Gotcha